java-backend/src/main/java/com/backend/javabackend/rest/UserRest.java

```diff
@@ -1,18 +1,82 @@
 package com.backend.javabackend.rest;
 
+import com.backend.javabackend.data.User;
+import com.backend.javabackend.data.UserResponse;
 import com.backend.javabackend.service.UserService;
 import lombok.RequiredArgsConstructor;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.time.LocalDateTime;
+import java.util.Map;
+
 import static com.backend.javabackend.constants.FullStackAppConstants.API_V_1_USER;
 
 @RestController
 @RequestMapping(value = API_V_1_USER)
 @RequiredArgsConstructor
 public class UserRest {
-    protected final Logger logger = LoggerFactory.getLogger(this.getClass());
+
     private final UserService userService;
+
+    @GetMapping("/users")
+    public ResponseEntity<UserResponse> getUsers() {
```

Comment:
https://medium.com/@nadinCodeHat/rest-api-naming-conventions-and-best-practices-1c4e781eb6a5

Reply:
Updated.